pkg/nsx/services/builder.go

```diff
@@ -0,0 +1,1286 @@
+package services
```

Comment:
do you want to call it firewall_builder.go, or the plan is to pull all structures in this file in the future?

Reply:
- firewall
   - builder.go
   - compare.go
   - ...
keep new CR in sub-directory, perhaps better.